packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -99,6 +99,12 @@ export function getProxyForField(field: FlexTreeField): TreeNode | TreeValue | u
 			// this case unreachable as long as users follow the 'array recipe'.
 			fail("'sequence' field is unexpected.");
 		}
+		case FieldKinds.identifier: {
```

Comment:
This is the same as the `FieldKinds.required` case. Is that right? We don't want `FlexFieldSchema<typeof FieldKinds.identifier>` or something? If it really is the same, then you can just have it share the same code as `FieldKinds.required`:

```ts
case FieldKinds.required:
case FieldKinds.identifier:
	const asValue = field as FlexTreeTypedField<
	// ...
```

Reply:
Updated to just return the unboxed value